src/pages/main.js

```diff
@@ -0,0 +1,9 @@
+import Form from '../components/Form';
+
+export default function Main() {
```

Comment:
```suggestion
export default function From() {
```
to avoid confusion i would recommend to name the pages and function after their respective functionalities :)